README.md

```diff
@@ -92,128 +77,18 @@ If interested in contributing or participating in the direction of KEDA, you can
 Just want to learn or chat about KEDA? Feel free to join the conversation in
 **[#KEDA](https://kubernetes.slack.com/messages/CKZJ36A5D)** on the **[Kubernetes Slack](https://slack.k8s.io/)**!
 
-## Building: Quick start with [Visual Studio Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers)
+## Become a listed KEDA user!
 
-This helps you pull and build quickly - dev containers launch the project inside a container with all the tooling
-required for a consistent and seamless developer experience.
+We are always happy to [list users](https://keda.sh/community/#users) who run KEDA in production, learn more about it [here](https://github.com/kedacore/keda-docs/blob/master/README.md#become-a-listed-keda-user).
```

Comment:
```suggestion
We are always happy to [list users](https://keda.sh/community/#users) who run KEDA in production, learn more about it [here](./README.md#become-a-listed-keda-user).
```
As we do below in L90, I think it's better to keep links pointing to the "current" branch